spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,19 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns a successful response' do
+      get posts_path
+      expect(response).to have_http_status(200)
+    end

```

Comment:
- Good job done so far, but this test is currently failing, please fix this 🙏. I suggest you use this instead `  expect(response).to have_http_status(:success)`
- Also, according to the project requirements you are required to test for all actions in the controller, I suggest you also test for the `post#show` action. Please fix this 🙏